Assets/Scripts/DrawLine.cs

```diff
@@ -262,8 +256,6 @@ private IEnumerator HideLine(string _lineId)
         yield return new WaitForSeconds(LINE_DURATION);
 
         if (transform.gameObject.tag == "Line" && lineId == _lineId) {
-            fingerPositions.Clear();
-            edgeCollider2D.points = new []{ new Vector2(), new Vector2() };
```

Comment:
What's the reason to remove these lines?

Reply:
I don't remember removing those lines can you please put it back sorry I might be changing it for experimental purpose and may have messed up I just changed the hard coded values